docs/guides/end-to-end-testing/migration/selenium-to-cypress.mdx

```diff
@@ -0,0 +1,396 @@
+
+One of the most significant changes between Selenium and Cypress is the actual testing syntax.
+While Selenium will run a class or different scenarios, Cypress will run every `it` in a `*.cy.js` file.
+So, the flow of a test should be transferred inside an `it`. To make matters more structured, it is possible to add
+several `it`s inside a `describe`. This allows you as the user to create a subdivision akin to test suites and test cases.
+
+```javascript
+describe('General information', () => {
+  beforeEach(() => {
+    cy.get('#loginButton').click()
+  })
+
+  it('Header is visible', () => {
+    cy.get('#header').should('be.visible')
+  })
+
+  it('Footer is visible', () => {
+    cy.get('#footer').should('be.visible')
+  })
+})
+```
+
+<br />
+Another big change is that you no longer need any WebDriver definitions. Cypress
+takes care of opening and closing the browser without any explicit prompting, unlike
+Selenium. Any code that is used to define the WebDriver is now obsolete.
+
+```java
+ WebDriver driver = new ChromeDriver();
+
+ WebDriverWait wait = new WebDriverWait(driver, 10);
+
+ driver.quit();
+```
+
+<br />
+Next would be to replace all the potential actions. The following code is fairly
+standard in Selenium frameworks.
+
+```python
+wait.until(EC.element_to_be_clickable((By.ID, "myButton"))).click()
+```
+
+<br />
+It is fairly easy to change this line of code to work inside a Cypress framework.
+It could even be done with a find and replace. In Cypress, the command would be the
+following:
+
+```Javascript
+cy.get('#myButton').click()
+```
+
+<br />
+Any wait until functionality that you have been using in Selenium is no longer required
+in Cypress. This functionality is [built in](/guides/core-concepts/retry-ability).
+
+The next vital part to migrate would be the [assertions](/guides/references/assertions). We can all agree assertions
+are moderately vital to a test automation framework. The next piece of code is a
+python Selenium assertion to validate if a button is visible. While it is possible
+to add third-party libraries, the built-in assertions of Cypress can tackle a wide
+range of situations.
+
+```python
+assert driver.find_element(By.ID, "myButton").is_displayed()
+```
+
+In Cypress, the same command would be:
+
+```javascript
+cy.get('#myButton').should('be.visible')
+```
+
+Considering the easy and clear syntax of Cypress, transferring all the selectors
+might take the longest. And while copying and pasting the selectors is definitely
+an option, rewriting the selectors might be required to conform to the best practices.
+
+##### iframe
+
+It's acknowledged that Cypress could handle iframes more effectively _out-of-the-box_. But, with the support of a robust
+community, solutions are readily available. In the case of iframes, the [cypress-iframe](https://www.npmjs.com/package/cypress-iframe)
+plugin was developed. This plugin will add a custom command to find and interact with an iframe.
```

Comment:
- PR https://github.com/cypress-io/cypress-documentation/pull/5789 from @dingraham
 proposes to deprecate the outdated [cypress-iframe](https://www.npmjs.com/package/cypress-iframe) plugin.